sns_ik_lib/matlab/snsIk_acc_rr_mt.m

```diff
@@ -0,0 +1,254 @@
+function [ddq, sData, exitCode] = snsIk_acc_rr_mt(ddqLow, ddqUpp, ddxGoalData, JData, dJdqData)
+% [ddq, sData, exitCode] = snsIk_acc_rr_mt(ddqLow, ddqUpp, ddxGoalDataData, JData, dJdqData)
+%
+% This function implements a generic multi-task version of the SNS-IK
+% acceleration solver that tries to meet multiple obJDataective terms
+% in a prioritized manner.
+%
+% INPUTS:
+%   ddqLow: lower limit for JDataoint accelerations
+%   ddqUpp: upper limit for JDataoint accelerations
+%   ddxGoalDataData: task-space accelerations of all the tasks in the order of priorities
```

Comment:
Same comment here as in the previous file: make the data structure clear for each input.